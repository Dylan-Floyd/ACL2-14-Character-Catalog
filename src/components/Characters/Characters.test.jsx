import { render, screen } from '@testing-library/react'
import { MemoryRouter } from 'react-router-dom/cjs/react-router-dom.min'
import Characters from './Characters.jsx'

it('should match the snapshot', () => {
  const charData = [
    {
      image: 'asdf.png',
      name: 'bob',
      species: 'man',
      status: 'alive'
    },
    {
      image: 'asdf2.png',
      name: 'bob2',
      species: 'man',
      status: 'alive'

    }
  ]
  const { container } = render(
    <MemoryRouter initialEntries={['/']}>
      <Characters characters={charData} />
    </MemoryRouter>
  )
  expect(container).toMatchSnapshot()
})