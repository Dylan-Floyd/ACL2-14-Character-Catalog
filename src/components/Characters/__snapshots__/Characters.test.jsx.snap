// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
<div>
  <ul
    aria-label="characters"
    class="list"
  >
    <li
      class="item"
    >
      <a
        href="/characters/undefined"
      >
        <figure>
          <img
            alt="bob"
            class="image"
            src="asdf.png"
          />
          <figcaption
            class="linkText"
          >
            bob
          </figcaption>
        </figure>
      </a>
    </li>
    <li
      class="item"
    >
      <a
        href="/characters/undefined"
      >
        <figure>
          <img
            alt="bob2"
            class="image"
            src="asdf2.png"
          />
          <figcaption
            class="linkText"
          >
            bob2
          </figcaption>
        </figure>
      </a>
    </li>
  </ul>
</div>
`;
